runtime/mount/agent.py

```diff
@@ -141,38 +141,50 @@ def _truncate_message_content(msg: MessageParam) -> MessageParam:
                 block_type = block.get("type")
 
                 if block_type in {"thinking", "redacted_thinking"}:
-                    truncated_blocks.append(block)
                     continue
 
                 if block_type == "tool_result":
                     result_str = block.get("content", "{}")
                     result = json.loads(result_str)
-
                     tool_name = result.get("tool_name")
-                    if tool_name in {"read_file", "grep"}:
-                        truncated_blocks.append(block)
-                        continue
 
-                    truncated_result = {
-                        "tool_name": tool_name,
-                        "success": result.get("success"),
-                        "summary": result.get("summary"),
-                    }
+                    if tool_name in {"create_cell", "create_markdown_cell"}:
```

Comment:
Do you think any of the returns mentioned in the tool descriptions for these should be updated? Like if `edit_cell` desc says "returns `original_code`" do you think it should be updated to indicate that the agent will not receive it (or if it's  a truncated field, that it will be truncated)?